types/detective/package.json

```
@@ -0,0 +1,6 @@
+{
+  "dependencies": {
+    "acorn": "^7.1.1"
```

```suggestion
    "acorn": "^7.0.0"
```
https://github.com/browserify/acorn-node/blob/v1.8.2/package.json#L10
(as per 1.8.2, not 2.*)